app/Kconfig

```diff
@@ -118,6 +118,12 @@ menuconfig ZMK_BLE
     select BT_SETTINGS
     select SETTINGS
 
+# BT_TINYCRYPT_ECC is required for BT_SMP_SC_PAIR_ONLY when using HCI
+if BT && !BT_CTLR
+config BT_TINYCRYPT_ECC
+    default y
+endif
```

Comment:
Any reason this can't be simplified?

```suggestion
config BT_TINYCRYPT_ECC
    default y if !BT_CTLR
```

Reply:
Because I don't know how Kconfig works? 😄 I'll try that.

Reply:
I tried that and it didn't work. Then I went back to what's in this commit, and that doesn't work either.

Reply:
Seems to just be build system weirdness. It works with a pristine build.